demisto_sdk/commands/lint/tests/test_linter/conftest.py

```diff
@@ -98,13 +98,13 @@ def _create_integration(content_path: Path, path: str = 'Integrations', no_lint_
         else:
             yml_dict = {}
             if js_type:
+                yml_dict['type'] = 'javascript'
                 if type_script_key:
                     yml_dict['script'] = {'type': 'javascript'}
-                yml_dict['type'] = 'javascript'
             else:
+                yml_dict['type'] = 'python'
```

Comment:
why do we need it twice

Reply:
What do you mean?
The first one is for javascript and the second one is for python

Reply:
i meant the yml_dict['script]['type'] = 'python'

Reply:
Oh actually I'm not really sure. 
I just switched between the lines, didn't want to break anything